x-pack/plugin/esql/qa/testFixtures/src/main/resources/ip.csv-spec

```diff
@@ -435,7 +435,7 @@ fe80::cae2:65ff:fece:feb9 | gamma
 
 implictCastingEqual
 required_feature: esql.string_literal_auto_casting_extended
-from hosts | where mv_first(ip0) == "127.0.0.1" | keep host, ip0;
+from hosts | where mv_first(ip0) == "127.0.0.1" | keep host, ip0 | sort host;
```

Comment:
Erm, I think this is still not deterministic in principle, because the host `beta` from our `host.csv` has 2 entries where the first `ip0` is `127.0.0.1`.

```suggestion
from hosts | where mv_first(ip0) == "127.0.0.1" | keep host, ip0 | sort host, ip0;
```

Reply:
Argh, nevermind - the produced rows for `beta` are identical...